no long double (yet), but we don't use that precision anyway!

--- origsrc/jewishcal-1.3/calctime.c	2002-05-13 13:09:39.000000000 -0500
+++ src/jewishcal-1.3/calctime.c	2006-06-13 14:58:42.609375000 -0500
@@ -268,7 +268,7 @@
           result = (double) phr * (double) phr_length;
 
           hour = (int) ((double) result / (double) 60.0);
-          min = (int) (fmodl((double) result, (double) 60.0));
+          min = (int) (fmod((double) result, (double) 60.0));
           iValue = hour*60 + min;
           
           lpCalc->iCurrentToken = CalcGetToken(lpCalc);
